finals-2024-01-16/code/5332-milana-sokolova/src/app/footer/footer.component.html

```diff
@@ -1,5 +1,9 @@
 <p style="border: dashed 1px blue;">
     Footer, put the student info here
+<p class="footer-content" style="border: dashed 1px blue;">
```

Comment:
The actual footer component is missing :smile:
And we also have the code twice...